FlipperZero/FlipperZero.xcodeproj/project.pbxproj

```diff
@@ -216,6 +223,16 @@
 			name = Frameworks;
 			sourceTree = "<group>";
 		};
+		498305C02527FE9E00B04AFD /* Fonts */ = {
+			isa = PBXGroup;
+			children = (
+				498305C22527FEE900B04AFD /* Born2bSportyV2.ttf */,
+				49F73FA5255FEEBF00402267 /* Roboto-Regular.ttf */,
```

Comment:
`Roboto-Regular.ttf` seems to be not included into all targets

Reply:
My bad.